dt-apps/dt-home/assets/css/home-screen.css

```diff
@@ -0,0 +1,1315 @@
+
+body.theme-dark .invite-modal-separator,
+.invite-modal-content.theme-dark .invite-modal-separator {
+    background: var(--border-color);
+}
+
+body.theme-dark .invite-url-input,
+.invite-modal-content.theme-dark .invite-url-input {
+    background: var(--surface-0);
+    border-color: var(--border-color);
+    color: var(--text-color);
+}
+
+body.theme-dark .invite-copy-button,
+.invite-modal-content.theme-dark .invite-copy-button {
+    background: var(--primary-color);
+}
+
+body.theme-dark .invite-copy-button:hover,
+.invite-modal-content.theme-dark .invite-copy-button:hover {
+    background: #3a8eef;
+}
+
+body.theme-dark .invite-close-button,
+.invite-modal-content.theme-dark .invite-close-button {
+    background: var(--surface-2);
+    border-color: var(--border-color);
+    color: var(--text-color);
+}
+
+body.theme-dark .invite-close-button:hover,
+.invite-modal-content.theme-dark .invite-close-button:hover {
+    background: var(--surface-2);
+    opacity: 0.9;
+}
+
+body.theme-dark .dt-launcher-bottom-nav {
+    background-color: var(--surface-0);
+    box-shadow: 0 -2px 10px var(--shadow-color);
+}
+
+body.theme-dark .dt-launcher-bottom-nav .nav-item:hover {
+    color: var(--primary-color);
+}
+
+body.theme-dark .dt-launcher-bottom-nav .nav-item-home,
+body.theme-dark .dt-launcher-bottom-nav .nav-item-home:hover {
+    background-color: var(--nav-home-bg-dark) !important;
+    color: var(--nav-text-white) !important;
+}
+
+body.theme-dark .dt-launcher-bottom-nav .nav-item-home i,
+body.theme-dark .dt-launcher-bottom-nav .nav-item-home:hover i {
+    color: var(--nav-text-white) !important;
+}
+
+body.theme-dark .dt-launcher-bottom-nav .nav-item-home:hover i {
+    opacity: 0.7;
+}
+
+body.theme-dark .dt-launcher-apps-selector {
+    background-color: var(--surface-0);
+    box-shadow: 0 -2px 10px var(--shadow-color);
+}
+
+body.theme-dark .dt-launcher-apps-selector li a {
+    color: var(--text-color);
+    border-bottom-color: var(--border-color);
+}
+
+body.theme-dark .dt-launcher-apps-selector li a:hover {
+    background-color: var(--surface-2);
+    color: var(--primary-color);
+}
+
+body.theme-dark .dt-launcher-apps-selector li a i:not([data-has-custom-color="true"]) {
+    color: var(--app-icon-default);
+}
+
+body:has(.dt-launcher-bottom-nav),
+#wrapper:has(.dt-launcher-bottom-nav) {
+    padding-bottom: var(--launcher-bottom-nav-height, 58px);
+}
+
+@media (max-width: 768px) {
+    .home-screen-container {
+        max-width: 100%;
+        width: 100%;
+    }
+    .dt-launcher-apps-selector ul {
+        margin: 0;
+        padding: 0 1rem;
+    }
+    .home-screen-container .apps-grid {
+        gap: 1rem 0.75rem;
+    }
+    .home-screen-header {
+        padding: 1rem 1.5rem;
+    }
+    .home-screen-header h1 {
+        font-size: 1.75rem;
+    }
+    .home-screen-header p {
+        font-size: 0.85rem;
+    }
+    .theme-toggle-button,
+    .menu-toggle-button {
+        width: 36px;
+        height: 36px;
+    }
```

Comment:
This won't take effect unless the selector matches the one further above: `.home-screen-container .theme-toggle-button, .home-screen-container .menu-toggle-button`. This one will take precedence then because it is further down the file.